protocol/app/upgrades/v5.0.0/upgrade.go

```diff
@@ -263,15 +264,17 @@ func voteExtensionsUpgrade(
 	keeper consensusparamkeeper.Keeper,
 ) {
 	currentParams, err := keeper.Params(ctx, &consensustypes.QueryParamsRequest{})
-	if err != nil {
+	if err != nil || currentParams == nil || currentParams.Params == nil {
 		panic(fmt.Sprintf("failed to retrieve existing consensus params in VE upgrade handler: %s", err))
 	}
-	if currentParams.Params.Abci.VoteExtensionsEnableHeight != 0 {
+	if currentParams.Params.Abci != nil && currentParams.Params.Abci.VoteExtensionsEnableHeight != 0 {
```

Comment:
The reason I initially put it in there is because if you've already set a VE enable height that's nonzero it's unknown (to me at least) what happens if you update that again. 

Happy to remove it since we assume this will be run once ever.